lib/AuthStateManager.ts

```diff
@@ -67,7 +67,7 @@ export class AuthStateManager {
     return this._authState;
   }
 
-  updateAuthState(): void {
+  async updateAuthState(): Promise<AuthState> {
```

Comment:
Should here be considered as a breaking change? Technically, we are exposing authStateManager from authClient (https://github.com/okta/okta-auth-js/blob/master/lib/OktaAuth.ts#L260)

Reply:
probably not a breaking change, as app code won't change because of this signature change.